### ISSUE TITLE
External links - appending text for screen readers 

### ISSUE THREAD

**Issue by jtart** [bbc/simorgh#428]
We need to apply text to external inline links that are read out by a screen reader. This text will be `, external`, i.e. `[TEXT], external`.

**Describe the solution you'd like**
Update the Inline Link component `@bbc/psammead-inline-link` to have a prop `isExternal` and when the value is `true`. This should add the correct Aria label: 

An aria-label should be used to indicate an external link to screen readers and the best support to achieve a pause is to use a comma in the code
```
<a href="https://www.bbc.co.uk/my-url" aria-label="Israel, external">Israel</a>
```

We should update the readme to add an explanation of this change in the Accessibility section. 

**Describe alternatives you've considered**
We considered adding Cypress integration tests for this. So a prerequisite for doing this work would be to update fixture data locally to include an article with an external link. However, we think that unit tests should be sufficient. 

**Testing notes**
Regression testing of the article using VoiceOver, TalkBack and on Desktop (JAWS).

Dev insight: 
- Unit tests should be added. 
- Use a screen reader to confirm that the label is read out as expected
https://bbc-news.github.io/accessibility-news-and-you/accessibility-and-testing-with-assistive-technology

**Additional context**

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

**Comment by greenc05** [comment]
An aria-label should be used to indicate an external link to screen readers and the best support to achieve a pause is to use a comma in the code
```<a href="https://www.bbc.co.uk/my-url" aria-label="Israel, external">Israel</a>```

**Comment by sareh** [comment]
How do we determine the link is `external`? Are we going to inspect the link itself, or will the CMS do this for us? 
If we are determining it ourselves, I suggest adding a function as part of this ticket: e.g. `isLinkExternal` which returns `true` or `false`.
- check if path is relative, if yes, then returns `false`
- check if first part of path is `https://www.bbc.co.uk` or `https://www.bbc.com` and if so, returns `false`
- otherwise returns `true`. 

Then using this function to determine whether we add the `, External` text to the offscreen text.

**Comment by sareh** [comment]
Blocked until we determine whether the CMS will return if the link is `external`.

**Comment by jtart** [comment]
@BogdanDogaru do we know this yet?

**Comment by BogdanDogaru** [comment]
@jtart we have a rough idea, but this all depends on the discussion over the format the rich text comes in. For now we can assume that we will have an indication in the feed for each external link (probably a boolean value).

**Comment by BogdanDogaru** [comment]
This is currently blocked pending a piece of work from our data aggregation layer to add a parameter for inline links whose host does not match `*.bbc.(co.uk|com)`

**Comment by jimjohnsonrollings** [comment]
The `model` for `urlLink` blocks now includes an `isExternal` boolean value that we can use to set this.

**Comment by sareh** [comment]
A prerequisite for doing this work is to update fixture data locally to include an article with an external link. This is so that we can add Cypress integration tests for this functionality.

**Comment by sareh** [comment]
Blocked pending an example existing in our fixture data.

**Comment by sareh** [comment]
After discussion, we decided that unit tests should be sufficient.

**Comment by ChrisBAshton** [comment]
@greenc05 

> An aria-label should be used to indicate an external link to screen readers and the best support to achieve a pause is to use a comma in the code

Is this because the alternative (`<a href="...">link text <span class="off-screen">, external</span></a>`) would make the screen reader user 'step' from "link text" to "external" rather than reading it as a collective unit?

If that's _not_ the case, I'd prefer the solution I've outlined above. From a technical perspective it's cleaner to keep the `InlineLink` component dumb and to have the _container_ pass the `<VisuallyHiddenText>, external</VisuallyHiddenText>` instead.

EDIT: Looks like my suggestion is ok according to BBCMAG: https://www.bbc.co.uk/guidelines/futuremedia/accessibility/mobile/links/descriptive-links.
